app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_split = command.split()
```

Comment:
better to use unpacking and save the parts into variables with  meaningful names. command_split[0], command_split[1] are not informative